components/benefit_sponsors/app/models/effective/datatables/benefit_sponsors_employer_datatable.rb

```diff
@@ -282,6 +282,10 @@ def nested_filter_definition
         end
         filters
       end
+
+      def authorized?(current_user, _controller, _action, _resource)
+        (current_user.present? && current_user.has_hbx_staff_role?) ? true : false
+      end
```

Comment:
No need for the extra operator,
`current_user.present? && current_user.has_hbx_staff_role?`
is fine on it's own.